metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/configs/HiveConnectorClientConfig.java

```diff
@@ -165,7 +174,7 @@ private HiveConf getDefaultConf(
         final int dataStoreTimeout = getDataStoreTimeout(connectorContext);
         result.setInt(HiveConfigConstants.JAVAX_JDO_DATASTORETIMEOUT, dataStoreTimeout);
         result.setInt(HiveConfigConstants.JAVAX_JDO_DATASTOREREADTIMEOUT, dataStoreTimeout);
```

Comment:
I am confused about the datastore_timeout and datastore_readtimeout. I think this place should use the  getDataStoreReadTimeout() function.

Reply:
Since hive metastore code is used for table/database creates and updates, and the timeout was earlier set to 60000, I used the same. The read and write are more for the read/writes of partitions and tables(deletes). I could have used the default timeout for the write in hive metastore conf but thought will try out the write timeout to be consistent across.